Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.BeLowerCased.cs

```diff
@@ -32,7 +32,7 @@ public void When_an_empty_string_is_supposed_to_be_lower_it_should_succeed()
         }
 
         [Fact]
-        public void When_a_non_lower_string_is_supposed_to_be_lower_it_should_fail()
+        public void When_an_upper_case_string_is_supposed_to_be_lower_it_should_fail()
```

Comment:
🔧 For the last two years or so, we have been adopting a fact-based naming convention, especially because of the parent class, we already know that we're testing `BeLowerCased` here. So in this case, we would name it something like `Upper_cased_alphabetic_characters_are_okay` or `Upper_cased_alphabetic_characters_are_valid`

Same for the other test cases.

Reply:
Okay